awx/api/serializers.py

```diff
@@ -4539,6 +4539,8 @@ def validate_rrule(self, value):
         try:
             Schedule.rrulestr(rrule_value)
         except Exception as e:
+            import traceback
+            logger.error(traceback.format_exc())
```

Comment:
When `dateutil` *does* blow up in an unexpected way, it's kind of nice to get some extra detail on why.